internal-docs/process/definition-of-done.md

```diff
@@ -0,0 +1,32 @@
+# Definition of Done
```

Comment:
Added a couple of the items from working agreements update to the components DoD, rest is the same